front/src/Settings.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react";
+
+import Turtle from "./Turtle";
+import TurtleEnableBar from "./TurtleEnableBar";
+
+/**
+ * Show the settings of all turtles.
+ * Author: H.E. van der Laan
+ *
+ * props:
+ * - turtles: an array of Turtles
+ */
+export default class extends Component {
+  disableTurtle(position) {
+    return state => {
+      state.turtles[position].enabled = false;
+      return state;
```

Comment:
hm, not sure about this.

Reply:
They are dead code now, should've removed them